packages/native-stack/package.json

```diff
@@ -53,7 +53,7 @@
     "react": "18.2.0",
     "react-native": "0.71.8",
     "react-native-builder-bob": "^0.21.0",
-    "react-native-screens": "~3.20.0",
+    "react-native-screens": "~3.25.0",
```

Comment:
need to use latest ver. with a types update: https://github.com/software-mansion/react-native-screens/pull/1846#issuecomment-1700507895